src/pages/Compound/useFarmBotRegistry.ts

```diff
@@ -6,39 +6,29 @@ import { AbiItem } from 'web3-utils'
 import { ERC20_ABI } from '../../constants/abis/erc20'
 import farmBotAbi from '../../constants/abis/FarmBot.json'
 import MOOLA_STAKING_REWARDS_ABI from '../../constants/abis/moola/MoolaStakingRewards.json'
-import { useFarmRegistry } from '../Earn/useFarmRegistry'
 
-export interface FarmBotSummaryBase {
+export interface FarmBotSummary {
   token0Address: string
   token1Address: string
   token0Name: string
   token1Name: string
   stakingTokenAddress: string
   totalLP: number
-}
-
-export interface FarmBotSummary extends FarmBotSummaryBase {
   address: string
   amountUserFP: number
   amountUserLP: number
   totalFP: number
   exchangeRate: number
   // userLPValue: number
   stakingRewardsAddress: string
-  rewardsUSDPerYear?: string
-  tvlUSD?: string
   totalLPInFarm: number
   totalLPSupply: number
 }
 
-export const farmBotAddresses = ['0xCB34fbfC3b9a73bc04D2eb43B62532c7918d9E81']
-
-export const useFarmBotRegistry = () => {
+export const useFarmBotRegistry = (farmBotAddresses: string[]) => {
```

Comment:
I'm getting a runtime error when running this locally on the Zap page; says that `farmBotAddresses` isn't iterable...

Reply:
oh no :( thanks for picking this up Joe! i did miss 2 components, the old compound component (unused) and the currency list - i guess this error must have been from there. i've updated! i forgot to check this scenario before, so glad you caught it

Reply:
i change my mind, the farmbot isn't needed on the currency list anymore (think this was used for the hackathon before we changed the UI) so i just removed the farmbot from there

Reply:
This is something we'll probably want again later at some point (for "zap betweens") but I think we can safely ignore that for the time being.